tests/IsNotEmptyTest.php

```
@@ -8,36 +8,28 @@
     {
         $this->assertFalse(Iterables::isNotEmpty([]));
         $this->assertTrue(Iterables::isNotEmpty(['foo']));
     }
 
-    /**
-     * @test
-     */
+    #[\PHPUnit\Framework\Attributes\Test]
     public function itReturnsFalseForEmptyArrayWithMixedKeys(): void
     {
         $this->assertFalse(Iterables::isNotEmpty([]));
```

PHPMD (2.15.0)

Avoid using static access to class '\Bonu\Iterable\Iterables' in method 'itReturnsFalseForEmptyArrayWithMixedKeys'. (undefined)

(StaticAccess)